app/src/main/AndroidManifest.xml

```diff
@@ -16,8 +16,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".NewsfeedActivity"
+            android:label="@string/title_activity_newsfeed"
+            android:theme="@style/AppTheme.NoActionBar" />
         <activity android:name=".Progress">
-            <intent-filter></intent-filter>
```

Comment:
@yusufelgharib  i had to remove the intent-filter tags since it was causing the project to not build